### DISCUSSION TITLE
Email rate limit exceeded?

### DISCUSSION THREAD

**Discussion by devhandler** [supabase/supabase#15880] [Questions] [answered]
was testing signup and tried 5, 6 times, which is definitely much lower than the limit specified by the free plan. anyone else having similar issues?

also wondering what's the best practice to mitigate abusers once you go production and have a paid plan? 
say, some users could sign up a lot of accounts or send many requests to your api or start huge amount of downloads from your storages.
ofc you can put a cap on your supabase plan/billing, but that will also stop other normal people from using your product. any practical solution here?

**Comment by GaryAustin1** [comment]
The limit was changed to 4 for signup a few days ago.
![image](https://github.com/supabase/supabase/assets/54564956/fedb129c-61bf-4974-b64d-4259830f6799)
https://status.supabase.com/

**Comment by ash-lewis** [comment]
We're currently having the same issue on the free plan during testing

**Reply by GaryAustin1** [reply, marked as answer]
https://github.com/orgs/supabase/discussions/15896

You will need to do your smtp service for more than very basic testing.  Signup type emails are limited to 4.

**Comment by devhandler** [comment]
thank you @GaryAustin1 . this is such a big change. curious if they could share some reasoning. maybe have to move to self hosting one day.

**Reply by GaryAustin1** [reply]
The link I last posted has the details.  SMTP from Supabase was always supposed to be limited to dev according to their production readiness docs.  Custom SMTP does not have this limit.
